### PR TITLE
Fixed Typographical Error in CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by iamrajiv**
#### Brief description of what is fixed or changed

Fixed Typographical Error in CODE_OF_CONDUCT.md

#### Release Notes

NO ENTRY

### CODE REVIEW

**Comment by iamrajiv** [comment]
can anyone please review this @sylee957 @asmeurer

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -48,13 +48,13 @@ threatening, offensive, or harmful.
...
 representing a project or community include using an official project e-mail
-address, posting via an official social media account, or acting as an appointed
+address, posting via an official social media account or acting as an appointed
 representative at an online or offline event. Representation of a project may be
 further defined and clarified by project maintainers.
 
 ## Enforcement
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing or otherwise, unacceptable behavior may be
```
Comment (smichr) [inline CODE_OF_CONDUCT.md]: no comma here where otherwise is an adjective for behavior.

File: release/README.md [hunk 2]
```diff
@@ -6,27 +6,26 @@ all release processes should be done with release.sh and rever.xsh.**
...
 First, make sure that you have done the following things
 
-- Create a release branch. Usually this branch is the same name as the release
+- Create a release branch. Usually, this branch is the same name as the release
   (e.g., "0.7.3"), although no naming convention is enforced on it.
 
 - Change the version in the release branch in sympy/release.py. If you want to
   do a release candidate, change it to a [PEP
   440](https://www.python.org/dev/peps/pep-0440) compliant version like
-  0.7.3rc1. Note that setuptools normalizes versions like 0.7.3.rc1 to
```
Comment (smichr) [inline release/README.md]: pluralization was correct; no change

File: release/README.md [hunk 3]
```diff
@@ -6,27 +6,26 @@ all release processes should be done with release.sh and rever.xsh.**
...
+  0.7.3rc1. Note that setuptools normalize versions like 0.7.3.rc1 to
   0.7.3rc1, so there will be errors if you do not use the latter form.
 
-- Change the version in master. This way, any additional changes made in master
```
Comment (smichr) [inline release/README.md]: no change necessary; latter use matches former at beginning of line

File: release/README.md [hunk 4]
```diff
@@ -54,9 +53,9 @@ script can take about an hour or so to run (depending on how long the tests
 take). Every time you re-run the script, it pulls from the branch and runs
 everything from scratch.
 
-At the end it will print two things, the list of authors, and the md5 sums.
```
Comment (smichr) [inline release/README.md]: I would leave "At" here and change the "things," to "things:" and remove the comma after "authors".

File: release/README.md [hunk 5]
```diff
@@ -54,9 +53,9 @@ script can take about an hour or so to run (depending on how long the tests
...
+In the end, it will print two things, the list of authors, and the md5 sums.
 Copy the list of authors into the release notes. You should verify that the
-md5 sums of the release files are the same as what are printed.
```
Comment (smichr) [inline release/README.md]: "...same as that which are printed."

**Comment by asmeurer** [comment]
The code of conduct comes from the contributor covenant, so any changes should be made upstream there. However, I believe the changes suggested here are all either incorrect, or correct either way (serial comma).

**Pull request closed without merging**